src/libraries/Native/Unix/System.Native/CMakeLists.txt

```diff
@@ -23,6 +23,12 @@ set(NATIVE_SOURCES
     pal_sysctl.c
 )
 
+if (CLR_CMAKE_TARGET_IOS)
+    set(NATIVE_SOURCES ${NATIVE_SOURCES} ios/pal_sys.m)
+else ()
+    set(NATIVE_SOURCES ${NATIVE_SOURCES} pal_console.c)
```

Comment:
pal_console.c is still in the original list of NATIVE_SOURCES, I assume you wanted to remove it there?

Reply:
oh, yes! thanks for noticing that